.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
I cannot find this script. Is one supposed to create this locally?

Reply:
@algomaster99 
I will update my PR as I think that this script is ignored. I will create an updated PR ASAP.